duktape/src/main/java/com/squareup/duktape/Duktape.java

```diff
@@ -63,6 +65,30 @@ public String evaluate(String script) {
   }
 
   /**
+   * Binds {@code object} to {@code name} for use in JavaScript as a global object.  {@type} defines
+   * the interface implemented by {@code object} that will be accessible to JavaScript.
+   */
+  public <T> void bind(Class<T> type, String name, T object) {
```

Comment:
I would flip the first two params here so the type and instance are next to each other. Plus that puts the name right after "bind" so it reads well:
```
duktape.bind("logger", Logger.class, logger);
```